src/App.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <div class="container mt-5">
+    <div class="d-flex justify-content-center">
+      <img class="logo" src="./assets/logo.svg" alt="logo" />
+      <h2 class="title">Todo App</h2>
+    </div>
+    <div class="d-flex justify-content-end">
+      <input v-model="taskName" type="text" class="form-control" placeholder="タスクを入力してください" />
+      <div class="ms-3"></div>
+      <button @click="submitTask" class="btn btn-primary primaryButton">
+        {{ editing ? "保存" : "追加" }}
```

Comment:
```suggestion
        {{ submitLabel }}
```

みたいなcomputedメソッドを定義するのも一手かも。（このままでも問題ないですが）

Reply:
なるほど！
htmlでロジックを持ちたくないので submitLabel を computed メソッドに作成しました！

https://github.com/yamatatsu10969/todo_vue/pull/1/commits/53d4008c47fe6f0b3d9836396040bc6c92306db9